coap-gateway/service/clientCreateHandler_test.go

```diff
@@ -38,8 +41,8 @@ func TestClientCreateHandler(t *testing.T) {
 
 	type args struct {
 		href          string
-		contentFormat message.MediaType
 		payload       []byte
+		contentFormat message.MediaType
```

Comment:
I see several structs have members reordered, to get smaller size?

Reply:
yes

Reply:
I want to go over all structs in repository but I want to it in separate PR